### DISCUSSION TITLE
Is there a way to start viewing the code diff without starting a formal review?

### DISCUSSION THREAD

**Discussion by digorithm** [pwntester/octo.nvim#354] [Q&A] [answered]
Sometimes I'd like to start browsing the changed files without starting a code review. That's because whenever I start a code review, a notification is sent to the PR itself:

<img width="343" alt="image" src="https://user-images.githubusercontent.com/3484025/211122122-0ca08d7b-ef27-4beb-b956-edb256447798.png">

Is there a way to start browsing the diff without `Octo review start`? Or prevent it from sending a notification to the PR?

**Comment by pwntester** [comment, marked as answer]
You can do `Octo pr changes`

**Comment by nikensss** [comment]
I actually have the same question. I find the view from `Octo review start` extremely useful. I always wanted to have something like that when I am just diffing my local changes with a different branch but got nothing as useful as that view. The proposed solution, `Octo pr changes`, is not as good as `Octo review start`.

Would it be possible to have `Octo pr changes` show the same view as `Octo review start`?

**Reply by zippeurfou** [reply]
I would like to double down on this. Being able to easily check the comments for example as you browse the code and browse file with `]q` is really helpful.
@nikensss did you ever find a solution?
